src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 30px $calender-padding;
+  gap: $date-cell-gap;
+  width: ($date-cell-side * 7) + ($date-cell-gap * 6) + ($calender-padding * 2);
+
+  @each $day, $date in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($date-cell-side * $date) + ($date-cell-gap * $date);
+    }
+  }
+
+  @for $lastDate from 28 through 31 {
+    &--month-length--#{$lastDate} {
+      .calendar__date:nth-child(n + #{$lastDate + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__date {
+    display: flex;
+    box-sizing: border-box;
```

Comment:
i think you already mentioned box-sizing in the _normalize.scss

Reply:
Right, Done!